extensions/everhour/src/components/TaskListItem.tsx

```diff
@@ -0,0 +1,67 @@
+import { List, ActionPanel, PushAction, Icon, Color, showToast, ToastStyle } from "@raycast/api";
+import { TimeSubmitForm } from "../views";
+import { Task } from "../types";
+import { startTaskTimer, stopCurrentTaskTimer } from "../api";
+
+export function TaskListItem({
+  task,
+  hasActiveTimer,
+  refreshActiveTimer,
+  fetchTasks,
+}: {
+  task: Task;
+  hasActiveTimer: boolean;
+  refreshActiveTimer: () => Promise<void>;
+  fetchTasks: () => Promise<void>;
+}) {
+  const enableTaskTimer = async () => {
+    try {
+      const { taskName } = await startTaskTimer(task.id);
+      refreshActiveTimer();
+      await showToast(ToastStyle.Success, "Timer started for " + taskName);
+    } catch (error) {
+      await showToast(ToastStyle.Failure, "Error starting timer");
+    }
+  };
+  const disableActiveTimer = async () => {
+    try {
+      const { taskName } = await stopCurrentTaskTimer();
+      refreshActiveTimer();
+
+      if (taskName) {
+        await showToast(ToastStyle.Success, "Timer stopped for " + taskName);
+      } else {
+        await showToast(ToastStyle.Failure, "No active timer");
+      }
+    } catch (error) {
+      await showToast(ToastStyle.Failure, "Error stopping timer");
+    }
+  };
+
+  const resolveTaskTime = (): string => {
+    const { timeInMin } = task;
+    if (timeInMin >= 60) {
+      const hours = Math.floor(timeInMin / 60);
+      const min = timeInMin % 60;
+      return `${hours} ${hours === 1 ? "hour" : "hours"} and ${min} min`;
+    } else {
+      return `${timeInMin} min`;
+    }
+  };
+
+  return (
+    <List.Item
+      id={task.id}
+      key={task.id}
+      title={`${task.name} - ${resolveTaskTime()}` + (hasActiveTimer ? " - Timer Active" : "")}
```

Comment:
I would suggest to not change the list item title for active timers. Instead tint the `icon` differently or add a `subtitle`. You can also play around with `List.Section` and have a section with active timers at the top of the list to give them priority.

```suggestion
      title={`${task.name} - ${resolveTaskTime()}`}
```